### DISCUSSION TITLE
no reports in ver4

### DISCUSSION THREAD

**Discussion by ariel256** [netbox-community/netbox#16161] [Help Wanted!]
Hi
since reports was removed, is there a way to define /edit /run new scripts with schedule? 
i.e. i have a script to check ping and send email to device contact when ping fail. how can i run this script every hour / 30min
and if there is a way, can it be in the GUI or only from the cli inside the docker?

**Comment by bitcollector1** [comment]
I currently have (4) scripts scheduled to run every day and (2) scripts running every 15 min. 

<img width="1298" alt="Screenshot 2024-05-16 at 09 28 39" src="https://github.com/netbox-community/netbox/assets/50723251/6273d5e6-0208-4aff-83ab-6a65ff5b33f3">

It's pretty simple to schedule a job from the GUI

<img width="1029" alt="Screenshot 2024-05-16 at 09 30 24" src="https://github.com/netbox-community/netbox/assets/50723251/594bcc9c-cd9a-4c94-b24a-81fbb18e6324">
